test/cases/compile_errors/asm_output_to_const.zig

```diff
@@ -0,0 +1,14 @@
+export fn foo() void {
+    const f: i64 = 1000;
+
+    asm volatile (
+        \\ movq $10, %[f]
+        : [f] "=r" (f),
+    );
+}
+
+// error
+// backend=llvm
+// target=native
```

Comment:
maybe your error is caught first so it doesnt matter, im not sure, but shouldnt the target be x86 since the test uses x86 asm?

Reply:
yeah this is testing a compile error so the asm is never run

Reply:
I didn't mean that it would run the asm, afaik the compiler will reject invalid assembly for whatever architecture it's being compiled for. but regardless still, doesn't matter in this case